gpt4all-chat/responsetext.cpp

```diff
@@ -633,7 +638,7 @@ void ResponseText::handleCodeBlocks()
         }
 
         cursor = mainFrame->lastCursorPosition();
-        cursor.setCharFormat(QTextCharFormat());
+        cursor.setCharFormat(textFormat);
```

Comment:
Hmm, this seems wrong here. Where is that defined?

Reply:
@manyoso Sorry I pushed this too early! I'll force push an update shortly (my first time writing in this language!)